index.php

```diff
@@ -1,6 +1,9 @@
 <?php declare(strict_types=1);
+include '../dev/log-to-console.php'; //! remove this and all logToConsole calls before merging with any other branches!
```

Comment:
Dit onder andere.

Reply:
removed the references to logtoconsole file